### ISSUE TITLE
A380-800 Map

### ISSUE THREAD

**Issue by Egan-31** [kolos26/GEOFS-LiverySelector#61]
Hi erveryone, during the time I designed the Singapore livery, I saved an intermediate version between Air France ad Singapore Airlines livery that is free of any logo. So If you want using it as a map for the Airbus A380, this is available.
![A380-800_Free](https://user-images.githubusercontent.com/115219384/194639664-d726a6fa-7fcf-4318-9a6f-f1614868fdf6.jpg)

**Comment by airgeorge** [comment]
thank you!

**Comment by Ryubunao** [comment]
cooool

**Comment by BoeingBus747** [comment]
Alright, sure is gonna help me

**Comment by iuhairways** [comment]
yesss

**Comment by Egan-31** [comment]
It's not perfect, but when you'll apply the airline compagny logo's and everything else, you wont see any difference. Just an advice, looking my little experience (of only one livery, yes I know), on the reactors, you can really see than even if i erased the air france logo, there is still a little marrk so try to put a logo there to hide the dot. It's what I did with the singapore, even if there is no logo there usually.

**Comment by BoeingBus747** [comment]
> It's not perfect, but when you'll apply the airline compagny logo's and everything else, you wont see any difference. Just an advice, looking my little experience (of only one livery, yes I know), on the reactors, you can really see than even if i erased the air france logo, there is still a little marrk so try to put a logo there to hide the dot. It's what I did with the singapore, even if there is no logo there usually.

Yeah, you can see that if you pay close attention but still useful tho

**Comment by kolos26** [comment]
Thank you very much, this will be useful
